templates/graph-config.xml.j2

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+       http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-2.5.xsd">
+   <bean id="gtfsBuilder" class="org.opentripplanner.graph_builder.impl.GtfsGraphBuilderImpl">
+        <property name="gtfsBundles">
+            <bean id="gtfsBundles" class="org.opentripplanner.graph_builder.model.GtfsBundles">
+                <property name="bundles">
+                    <list>
+                        <!--
```

Comment:
Update: Since there's already a bunch of stuff in this PR, I'll make the gtfs import a separate PR after this one.